gwswlib/hydx.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Hydx:
+    def __init__(self):
+        self.connection_nodes = []
+
+
+class ConnectionNode:
+    MAAIVELDSCHEMATISERINGCOLL = [
+        {"Label": "Reservoir", "Code": "RES"},
+        {"Label": "Gekneveld", "Code": "KNV"},
+        {"Label": "Verlies", "Code": "VRL"},
+    ]
+
+    MATERIAALHYDXCOLL = [
+        {"Label": "Beton", "Code": "BET"},
+        {"Label": "Gewapend beton", "Code": "BET"},
+        {"Label": "Polyvinylchloride", "Code": "PVC"},
+        {"Label": "Gres", "Code": "GRE"},
+        {"Label": "Gietijzer", "Code": "GIJ"},
+        {"Label": "Metselwerk", "Code": "MSW"},
+        {"Label": "Metselwerk (baksteen)", "Code": "MSW"},
+        {"Label": "Metselwerk (bepleisterd)", "Code": "MSW"},
+        {"Label": "Metselwerk (onbepleisterd)", "Code": "MSW"},
+        {"Label": "HDPE", "Code": "HPE"},
+        {"Label": "Polyester", "Code": "HPE"},
+        {"Label": "Polyetheen", "Code": "HPE"},
+        {"Label": "Polypropyleen", "Code": "HPE"},
+        {"Label": "Plaatijzer", "Code": "PIJ"},
+        {"Label": "Staal", "Code": "STL"},
+    ]
+
+    VORMPUTCOLL = [
+        {"Label": "Rechthoekig", "Code": "RHK"},
+        {"Label": "Rond", "Code": "RND"},
+    ]
+
+    TYPEKNOOPPUNTCOLL = [
+        {"Label": "Inspectieput", "Code": "INS"},
+        {"Label": "Infiltratieput", "Code": "ITP"},
+        {"Label": "Compartiment", "Code": "CMP"},
+        {"Label": "Uitlaat", "Code": "UIT"},
+    ]
+
+    STATUSOBJECTCOLL = [
+        {"Label": "In gebruik", "Code": "ACT"},
+        {"Label": "In ontwerp", "Code": "ONT"},
+    ]
+
+    AANNAMEHYDXCOLL = [
+        {"Label": "Expert judgement", "Code": "EXJ"},
+        {"Label": "Conform ontwerp", "Code": "ONT"},
+        {"Label": "Conform norm", "Code": "NRM"},
+    ]
+
+    FIELDS = [
+        {
+            "csvheader": "UNI_IDE",
+            "fieldname": "HydxIdentificatieKnooppuntOfVerbinding",
+            "unit": "-",
+            "type": ('string', 10),
```

Comment:
`"string"` is een "magische string": een lapje tekst die heel specifieke betekenis heeft omdat er elders op gecheckt wordt. Je kan dan beter een constante gebruiken. (`STRING_TYPE = "string"` en dan `STRING_TYPE` gebruiken).

Reply:
Deze snap ik niet helemaal. Wat bedoel je?